primitives/src/storage_keys.rs

```diff
@@ -33,3 +34,9 @@ pub enum NftsKeys {
 	/// Get the attribute value of `item` of `collection` corresponding to `key`.
 	CollectionAttribute(CollectionId, BoundedVec<u8, KeyLimit>),
 }
+
+#[derive(Encode, Decode, Debug, MaxEncodedLen)]
+pub enum TrustBackedAssetsKeys {
+	/// Check if the asset exists.
+	AssetExists(AssetId),
```

Comment:
This only has one option for reading

Reply:
This is partially due to https://github.com/r0gue-io/pop-node/issues/26